### ISSUE TITLE
WMS 1.3.0 server not sending 'Content-Type' tag in header causes KeyError

### ISSUE THREAD

**Issue by JaviBecerra** [geopython/OWSLib#564]
Hi,
TLDR: server responses without 'Content-Type' seem to be allowed by HTTP1.1, and wms130.py should manage the case nicely instead of throwing a KeyError exception.

We are connecting to a public WMS server that shows a couple unusual behaviours:
-It works fine when QUERY_LAYERS parameter is all caps (i.e. a request with ...&QUERY_LAYERS=...) works fine, but exactly the same request with query_layers parameter in lowercase fails (...&query_layers=...). This is not compliant to WMS specification, and the organization managing the server is aware of it. This uppercase restriction does not seem to apply to any other parameter on the request, only to QUERY_LAYERS. (I would not expect the library to manage this non-compliant cases unless it is a common case, which I doubt, just here FYI and so that you know that if you try connecting to the server with owslib it will fail unless you manually change the QUERY_LAYERS parameter in request to uppercase...).
-It seems that the request does not return a 'Content-Type' tag, making OWSLib to crash with KeyError on wms130.py line 130. Once again, I do not know if this is specific to this server or appears more often (to my surprise it seems that returning a Content-Type is not mandatory as per HTTP 1.1 https://stackoverflow.com/questions/15860742/is-content-type-mandatory-in-http-post-request). Handling this could easily be done adding a ```try:/Except KeyError: pass``` block to wms130.py `if` block on line 390.
A sample request for this specific WMS server:
http://wms.mapama.gob.es:80/sig/Agua/EvalRecHidricos/1940_2005/ETP/wms.aspx?service=WMS&version=1.3.0&request=GetFeatureInfo&layers=Evapotranspiraci%C3%B3n+potencial+anual+%281940%2F41-2005%2F06%29&styles=&width=3&height=3&crs=EPSG%3A25830&bbox=541426.7233056404%2C4704581.837303154%2C541681.7233056404%2C4704836.837303154&format=image%2Fpng&transparent=FALSE&bgcolor=0xFFFFFF&exceptions=XML&QUERY_LAYERS=Evapotranspiraci%C3%B3n+potencial+anual+%281940%2F41-2005%2F06%29&i=1&j=1&info_format=application%2Fvnd.ogc.wms_xml&feature_count=1
Cheers
